transcom/mymove
# B-21566 INT

## [Agility ticket]
(https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1023571&RoomContext=TeamRoom%3A1007470&concept=TeamRoom)

## Summary
This story is to support the new UBP service item which is replacing these now obsolete service items: ICOUB, IOOUB, IOCUB, NSTUB. The obsolete services are removed from service_params, and re_services. All rows for pricing of the obsolete services in re_intl_prices are now pointed to the new UBP service. The re_service_items table is added to associate a service to a market/shipment type combo.

Is there anything you would like reviewers to give additional scrutiny? Maximum scrutiny on all changes.

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

###Database

1. Run the migrations.
2. Verify that IOCUB, ICOUB, IOOUB, and NSTUB are no longer in the re_services table.
3. Verify that there are no rows tied to IOCUB, ICOUB, IOOUB, and NSTUB in the service_params table.
4. Verify that there are no rows tied to  IOCUB, ICOUB, IOOUB, and NSTUB in the re_intl_prices table.
5. Verify that there are rows tied to UBP in the re_intl_prices table.
6. Verify that there are rows tied to the UBP service in the re_service_items_table.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
